Extension/gulpfile.js

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+const gulp = require('gulp');
+const tslint = require('gulp-tslint');
+const mocha = require('gulp-mocha');
+
+gulp.task('unitTest', () => {
+    gulp.src('./out/test/unitTests', {read: false}).pipe(
+        mocha({
+            ui: "tdd"
+        })
+    );
+});
+
+/// Misc Tasks
+const allTypeScript = [
+    'src/**/*.ts',
+    '!**/*.d.ts',
+    '!**/typings**'
+];
+
+const lintReporter = (output, file, options) => {
+    //emits: src/helloWorld.c:5:3: warning: implicit declaration of function ‘prinft’
+    var relativeBase = file.base.substring(file.cwd.length + 1).replace('\\', '/');
```

Comment:
see if you could get source navigation from the lint output to code, from vscode build output.